README.md

```diff
@@ -15,8 +15,28 @@ The gulpfile is 80% from the [gulp-webapp](https://github.com/yeoman/generator-g
 
 There are two commands, both run via npm.
 
-- `npm run build` - clean & build everything and put it into dist folder
-- `npm run serve` - serve the pages and utilize live reload on changes to styles, fonts, images, scripts and HTML.
+- `npm run build` or `gulp build` or `gulp` - clean & build everything and put it into dist folder
```

Comment:
Ah... I'm remembering now -- I think grunt does the same thing: when you `npm install -g` grunt or gulp, you're really just installing a thin cli that looks for a local one.  Presumably it's designed this way precisely because of the versioning issue @jflasher mentioned.